frontend/src/main/pages/HomePage.css

```diff
@@ -16,9 +21,14 @@
   text-fill-color: transparent;
   -webkit-background-clip: text;
   -webkit-text-fill-color: transparent;
-  animation: textclip 2s linear infinite;
+  animation: textclip 8s linear infinite;
   display: inline-block;
   font-size: 50px;
+  font-family: 'Rye', cursive;
+}
+.reg-text 
```

Comment:
⛏️ I tried Ctrl+F'ing this section, and it doesn't seem like this CSS portion is used. Maybe take it out if it doesn't affect anything?